node/typings/Search.ts

```diff
@@ -107,3 +99,93 @@ interface Breadcrumb {
   href: string
   name: string
 }
+
+interface BiggySearchProduct {
```

Comment:
can we rename these types without the `Biggy` prefix or would they conflict with types that already exist?

Reply:
Some of them would conflict. Even then, I like some sort of prefix (might be not biggy), but as to differ which DTO is the one that comes from the biggy DB, and the one that goes out.